ci/test_suites/authenticators_gcp/get_tokens_to_files.sh

```diff
@@ -59,9 +59,9 @@ _get_token_to_files() {
     local token=""
 
     if [ -z "$auth_token" ]; then
-      token="$(curl -s -H 'Metadata-Flavor: Google' $token_url)" || exit 1
+      token="$(curl --location --silent -H "Metadata-Flavor: Google" $token_url)" || exit 1
```

Comment:
does this work? shouldn't it be `-H 'Metadata-Flavor: Google'`? because the line starts with `"`.